utopia-remix/app/routes/projects.tsx

```diff
@@ -860,6 +925,8 @@ const ActiveOperation = React.memo(({ operation }: { operation: Operation }) =>
         return 'Renaming'
       case 'restore':
         return 'Restoring'
+      case 'changeAccess':
+        return 'Changing access for'
```

Comment:
as a heads up, this has been changed so you'll probably need to resolve some conflict when merging w/ master after https://github.com/concrete-utopia/utopia/pull/4979 is merged